js/modules/Helpdesk/HelpdeskConfigController.js

```diff
@@ -0,0 +1,263 @@
+            const url_params = new URLSearchParams({
+                profile_id: this.#profile_id,
+            });
+            const response = await fetch(`${url}?${url_params}`);
+            if (!response.ok) {
+                throw new Error(response.status);
+            }
+
+            this.#getTilesContainerDiv().innerHTML = await response.text();
+        } catch (e) {
+            glpi_toast_error(__('An unexpected error occurred.'));
+            console.error(e);
+        }
+    }
+
+    async #saveTilesOrder()
+    {
+        try {
+            // Set up form data
+            const form_data = new FormData();
+            form_data.append('profile_id', this.#profile_id);
+            this.#getTilesOrder()
+                .forEach((id) => form_data.append("order[]", id))
+            ;
+
+            // Send request
+            const url = `${CFG_GLPI.root_doc}/ajax/Config/Helpdesk/SetTilesOrder`;
+            const response = await fetch(url, {
+                method: 'POST',
+                body: form_data,
+                headers: {
+                    'X-Glpi-Csrf-Token': getAjaxCsrfToken(),
+                }
+            });
+
+            // Handle server errors
+            if (!response.ok) {
+                throw new Error(response.status);
+            }
+
+            // Refresh content and confirm success
+            this.#getTilesContainerDiv().innerHTML = await response.text();
+            glpi_toast_info(__("Configuration updated successfully."));
+        } catch (e) {
+            glpi_toast_error(__('An unexpected error occurred.'));
+            console.error(e);
+        }
+    }
+
+    #getTilesOrder()
+    {
+        const nodes = this.#container
+            .querySelectorAll('[data-glpi-helpdesk-config-tile-profile-id]')
+        ;
+
+        return [...nodes].map((node) => {
+            return node.dataset.glpiHelpdeskConfigTileProfileId;
+        });
+    }
+
+    #getTilesContainerDiv()
+    {
+        return this.#container.querySelector("[data-glpi-helpdesk-config-tiles");
+    }
+
+    async #deleteTile(tile_container)
+    {
+        // Hide content immediatly (optimistic UI)
+        tile_container.classList.add('d-none');
+
+        try {
+            const tile = tile_container.querySelector('[data-glpi-helpdesk-config-tile]');
+
+            // Set up form data
+            const form_data = new FormData();
+            form_data.append(
+                'tile_id',
+                tile.dataset.glpiHelpdeskConfigTileId
+            );
+            form_data.append(
+                'tile_itemtype',
+                tile.dataset.glpiHelpdeskConfigTileItemtype
+            );
+
+            // Send request
+            const url = `${CFG_GLPI.root_doc}/ajax/Config/Helpdesk/DeleteTile`;
+            const response = await fetch(url, {
+                method: 'POST',
+                body: form_data,
+                headers: {
```

Comment:
Will be necessary for #18723.

```suggestion
                headers: {
                    'X-Requested-With': 'XMLHttpRequest',
```